.travis.yml

```diff
@@ -84,24 +84,24 @@ install:
 script:
   - mkdir -p target/"$TARGET";
   - >
-      case "$TARGET" in
-        *-linux-android*) DOCKER=android   ;; # Android uses a local docker image
-        *-apple-darwin)                    ;;
-        *)                DOCKER="$TARGET" ;;
-      esac;
-      if [ -n "$DOCKER" ]; then
-        sh ci/build-run-docker.sh "$DOCKER" "$TARGET" "$SKIP_TESTS";
-      else
-        sh ci/run.sh;
-      fi
+    case "$TARGET" in
+      *-linux-android*) DOCKER=android   ;; # Android uses a local docker image
+      *-apple-darwin)                    ;;
+      *)                DOCKER="$TARGET" ;;
+    esac;
+    if [ -n "$DOCKER" ]; then
+      sh ci/build-run-docker.sh "$DOCKER" "$TARGET" "$SKIP_TESTS";
+    else
+      sh ci/run.sh;
+    fi
 
   # Check the formatting last because test failures are more interesting to have
   # discovered for contributors lacking some platform access for testing beforehand
   - if [ "${TARGET}" = x86_64-unknown-linux-gnu ]; then
-      shellcheck -s dash -e SC1090 -- rustup-init.sh ci/*.sh;
-      rustup component add rustfmt;
-      rustfmt -vV;
-      cargo fmt --all -- --check;
+    shellcheck -s dash -e SC1090 -- rustup-init.sh ci/*.sh;
```

Comment:
Are those reflow necessary?  IMO it is harder to see after this change.

Reply:
This one is bad, I'll fix it.